static/src/components/builder/store/modules/multiselectInput.js

```diff
@@ -42,7 +42,7 @@ export const getters = {
       labelAdded: state.labelAdded,
       choices: state.choiceList.map(choice => choice.value.trim()),
       pybAnswer: state.pybAnswer,
-      initialValue: state.initialValue,
+      initialValue: (state.initialValue.length > 0) ? [state.initialValue] : null,
```

Comment:
Can `state.initialValue` ever be `None`? If so, this check could raise an exception when accessing  the `length` property. Also, I believe if we initialize the value to `null` you could just set this property as-is: `initialValue: state.initialValue`

Reply:
I think the value comes in as an empty string which is why it needs to be overwritten to null so that the blue line isn't shown for the empty string.

Reply:
Will the value still come in as an empty string if it's now being initialized to an empty array? *May need to confirm this.*

If the value is indeed now coming as an array, then re-casting it to an array with `[state.initialValue]` would create a double-array `[[]]`.

In either case, perhaps we should include a check to make sure `state.initialValue` is not null before checking its length property.

```js
initialValue: (state.initialValue && state.initialValue.length > 0) ? [state.initialValue] : null
```

Reply:
I followed @ywrsusan 's  suggestion and no longer needed this logic 👍